Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -96,11 +96,12 @@ def get_server_url(server_url: str) -> str:
     return url
 
 
-def create_ticket_context(data: dict) -> Any:
+def create_ticket_context(data: dict, additional_fields: list = []) -> Any:
```

Comment:
I don't think you can do that, it should be `None` in the decleration.

Reply:
![image](https://user-images.githubusercontent.com/35098543/80069702-7843ba00-854a-11ea-9332-dae24030eb41.png)